Realm/Tests/DictionaryPropertyTests.m

```diff
@@ -1667,4 +1667,29 @@ - (void)testSetDictionaryManaged {
     [dict.realm cancelWriteTransaction];
 }
 
+- (void)testInitWithNullLink {
```

Comment:
This could probably use a test in Swift too.

Reply:
Added a test.